src/App.jsx

```diff
@@ -1,11 +1,18 @@
 import Bio from "./components/Bio";
+import { useState } from "react";
+import Navbar from "./components/Navbar";
+import NavModal from "./components/NavModal";
 
 const App = () => {
+
+  const [isShowing, setIsShowing] = useState(false);
```

Comment:
`isShowing` var name should reference what's showing. So like modalShowing is isMobileVisible